app/views/content/train-to-be-a-teacher/promos/_find-your-undergraduate-course.html.erb

```diff
@@ -6,7 +6,7 @@
   </div>
 
   <div class="right">
-    <span class="caption-l caption--bold caption--purple">Find your course</span>
+    <h2 class="heading-m">Find your teacher training course</h2>
```

Comment:
@ethax-ross I've changed the captions here to headings (for accessibility reasons) but now all the text has shifted down and is no longer aligned with the top of the icon on the left - would you be able to take a look pls?

Reply:
I've updated these